jekyll/_cci2/jenkins-converter.md

```diff
@@ -0,0 +1,165 @@
+
+## Supported Syntax
+
+Only declarative (pipeline) `Jenkinsfile`s are currently supported.
+
+| Jenkinsfile Syntax | Approx. CircleCI Syntax                                                                          | Status                                                                                |
+| ------------------ | ------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------- |
+| agent              | [executor](https://circleci.com/docs/2.0/configuration-reference/#executors-requires-version-21) | Static                                                                                |
+| post               | [when attribute](https://circleci.com/docs/2.0/configuration-reference/#the-when-attribute)      | See [when](https://circleci.com/docs/2.0/configuration-reference/#the-when-attribute) |
+| stages             | [workflows](https://circleci.com/docs/2.0/workflows/)                                            | Supported                                                                             |
+| steps              | [step](https://circleci.com/docs/2.0/jobs-steps/#steps-overview)                                 | Limited                                                                               |
+| environment        | [environment](https://circleci.com/docs/2.0/env-vars/)                                           | [Unsupported](https://github.com/circleci/jenkinsfile-converter/issues/26)                                                                           |
+| options            | N/A                                                                                              | See [Supported Jenkins Plugins](#supported-jenkins-plugins)                           |
+| parameters         | [parameters](https://circleci.com/docs/2.0/reusing-config/#using-the-parameters-declaration)     | Unsupported                                                                           |
+| triggers           | [cron](https://circleci.com/docs/2.0/workflows/#scheduling-a-workflow)                           | Unsupported                                                                           |
+| stage              | [job](https://circleci.com/docs/2.0/configuration-reference/#jobs)                               | Supported                                                                             |
+
+## Supported Jenkins plugins
+
+**Important Note: Jenkinsfiles relying on plugins not listed below cannot be converted**. Please remove stanzas relying on those unsupported plugins (e.g. `options`), otherwise <u>you will see an error message saying something is "Unknown" or "Invalid"</u>. Please submit a ticket with our support center if you have a request to add a plugin to the list.
```

Comment:
```suggestion
**Note: Jenkinsfiles relying on plugins not listed below cannot be converted**. Please remove stanzas relying on those unsupported plugins (for example `options`), otherwise <u>you will see an error message saying something is "Unknown" or "Invalid"</u>. Please submit a ticket with our support center if you have a request to add a plugin to the list.
```